packages/plugin-ext/src/main/browser/view/plugin-view-widget.ts

```diff
@@ -138,6 +150,38 @@ export class PluginViewWidget extends Panel implements StatefulWidget, Descripti
         this.onDidChangeDescriptionEmitter.fire();
     }
 
+    get badge(): number | undefined {
+        if (this._badge === undefined) {
+            for (const w of this.widgets) {
```

Comment:
This algorithm seems strange. Other places seem to assume that the "wrapped" widget is `this.widgets[0]`. Why is the algorithm for `badge` different from `descritption`?

Reply:
That is indeed more coherent here with `this.widget[0]`.
This is adressed in the new version.